### PR TITLE
corrected grammatical typos and errors

### PR DESCRIPTION
**Pull request by Dishebh**
Corrected some grammatical typos and errors in this PR.

### CODE REVIEW

File: community/index.md [hunk 1]
```diff
@@ -149,9 +149,9 @@ the community.
 - stripes-loader: Jason Skomorowski (lead)
 - stripes-sample-platform: Jason Skomorowski (lead)
 - okapi-stripes: Wolfram Schneider (lead)
-- ui-users: Mike Taylor, Niels Erik Nielsen, Jason Skomorowski, John Coburn, Matt Connolly, Jeremy Huff
-- ui-items: Mike Taylor
-- ui-okapi-console: Mike Taylor, Niels Erik Nielsen
+- UI-users: Mike Taylor, Niels Erik Nielsen, Jason Skomorowski, John Coburn, Matt Connolly, Jeremy Huff
+- UI-items: Mike Taylor
+- UI-okapi-console: Mike Taylor, Niels Erik Nielsen
 - cql2pgjson-java: Julian Ladisch (lead)
```
Comment (dcrossleyau) [inline community/index.md:155]: Please do not change the names of modules. They are explicitly named with lower-case.
See https://dev.folio.org/source-code/map/#ui

File: source-code/index.md [hunk 2]
```diff
@@ -27,15 +27,15 @@ A good starting point for understanding the FOLIO code is
...
+applications running on the FOLIO platform and enable access to other modules
 in the architecture.
 
 The FOLIO system is made up of the code in several GitHub repositories.
 Each repository contains the code for a single well-defined element of the
 system. These repositories fall into three categories:
 
 - _server-side elements_ that provide services and the
-  infrastructure that they run on;
+  the infrastructure that they run on;
```
Comment (dcrossleyau) [inline source-code/index.md:38]: No, that now duplicates "the the".

File: source-code/index.md [hunk 3]
```diff
@@ -62,12 +62,10 @@ facilitated by the code in the `raml-module-builder` repository.
...
 - [raml](https://github.com/folio-org/raml)
-  -- Repository of RAML files, including JSON Schemas, traits and
-  resource types centralized for re-usability.
-  The [API reference](/reference/api/) documentation is also
+  -- Repository of RAML files, including JSON Schemas, traits and resource types centralized for re-usability.
+  The [API Reference](/reference/api/) documentation is also
   generated.
-  This repository is the master location for the traits and resource
-  types, while each module is the master for its own schemas, examples,
+  This repository is the master location for the traits and resource types, while each module is the master for its schemas, examples,
```
Comment (dcrossleyau) [inline source-code/index.md:68]: Please do not join lines together. That just makes the "diff" harder to review.

**Comment by Dishebh** [comment]
@dcrossleyau thank you some for the review! I did some changes as you asked for, pls have a look at them as well.

**Pull request merged**